p2/test2.py

```diff
@@ -0,0 +1,30 @@
+from p2 import Problem2
+
+class Test2:
+    def test_element_at_equals(self):
+        assert Problem2([1,2,3,4]).element_at(0)==1
+    def test_element_at_index_bigger_len_arr(self):
```

Comment:
leave a line between each method, it doesn't affect the code compilation but increases the readability.